incubator-tools/cer_wer/cer_wer.ipynb

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #79.        print(f"Mean CER = {round(mean_cer,2)}%, Mean WER = {round(mean_wer,2)}%")
</pre><p>You can replace the round functions with <code>{mean_cer:.2f}</code> and  <code>{mean_wer:.2f}</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Acknowledged.</p>